packages/open-truss/src/configuration/engine-v1/Frame.tsx

```diff
@@ -247,7 +252,12 @@ function processProps({
       const signalName = parseSignalName(viewPropVal) ?? ''
       signal = signals[signalName]
     } else if (viewPropVal !== undefined) {
-      signal = signalsType.parse(viewPropVal)
+      // TODO This condition is for viewProps that are hard coded values (not signals)
+      // We need to transform them into a signal so that the component can use them.
+      // If creating many signals becomes an issue, perhaps we can make a simple wrapper
+      // object that mirrors the signal interface (eg. signal.value) and use that instead
+      signal = signalsType.parse(undefined)
```

Comment:
@kmcq the update to the signals code you made seem to break the code for passing in hardcoded view props. eg.

```yaml
props:
  title: Hello world
```

Previously we were doing `signal = signalsType.parse(viewPropVal)`, but the new validator logic you added causes this to throw an exception because hardcoded values fail `isSignalLike` as they are scalars / primitives. Instead parsing `undefined` gives us the default object and we just set the view prop value accordingly.